src/app/chat/app-container/chat-container/chat-container.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-chat-container',
+  templateUrl: './chat-container.component.html',
+  styleUrls: ['./chat-container.component.scss']
```

Comment:
use OnPush https://www.thinktecture.com/en/angular/whats-the-hype-onpush/#:~:text=The%20OnPush%20strategy%20changes%20Angular's,a%20subtree%20for%20that%20component.